### ISSUE TITLE
Make channel joining UI more clear

### ISSUE THREAD

**Issue by GoogleFrog** [ZeroK-RTS/Chobby#257]
The channel joining UI is unclear, apparently. Here are some suggestions:
 * Add a tooltip for the '+' button.
 * Replace the text of the '+' channel with 'Join'.
 * Make /j and /join typed in chat join channels. People are used to this UI.

**Comment by gajop** [comment]
Do second and third imo

**Comment by DeinFreund** [comment]
Maybe add a drop down list of common channels. For example #zktourney or #zkdev.

**Comment by gajop** [comment]
That's https://github.com/Spring-Chobby/Chobby/issues/87 imo

**Comment by Licho1** [comment]
I would go with 1 and 3 .. reason is that channel joining is a "pro" function.
And having a button with join makes it look like important feature, like join a game.
Other games also rely on simple icons/buttons (think overwatch) even fore more important stuff.

**Comment by gajop** [comment]
old dota2 (just has the "+"): http://i.imgur.com/JAxilKw.jpg
new dota2 (has "Channels +"): https://i.ytimg.com/vi/umgRkDfz8Sw/maxresdefault.jpg

So both options work. I don't know why they chose to go with Caption and icon, but they seem to have done it consistently (see balanced shuffle, swap teams, leave this lobby, etc.)

I guess the really important thing here is 3)

**Comment by Licho1** [comment]
Perhaps 3 could be implement server side? So it works universally anywhere>

**Comment by gajop** [comment]
Yes, it should be. There used to be uberserver code that would ask the server for each channel's name and number of users inside, and would then sort the list in descending order and allow joining by clicking on a button. That code is now gone (it was also resource intensive as there can be many channels), but a similar solution could be implemented, using a popup window instead.

I suggest implementing a server response for a user query command that would list top N most populated channels, where N is 20 or something. If you want to go through the extra trouble you can implement featured channels or create some sort of paging support for this.

**Comment by Licho1** [comment]
Oh what I meant was the handling of "/j " commands.. regarding chat list.. I dont know..its all so obscure..

**Comment by gajop** [comment]
I don't think any / commands should be implemented on the server.

**Comment by DeinFreund** [comment]
> Perhaps 3 could be implement server side? So it works universally anywhere>
This wouldn't be very universal as it's common for lobbies to filter out messages prefixed with `/`.
